server/pulp/server/event/http.py

```diff
@@ -25,56 +24,36 @@
 def handle_event(notifier_config, event):
     # fire the actual http push function off in a separate thread to keep
     # pulp from blocking or deadlocking due to the tasking subsystem
-
     data = event.data()
-
     _logger.info(data)
-
-    body = json.dumps(data, default=json_util.default)
```

Comment:
python-requests handles it all - we just have to hand it a `dict` rather than a string of JSON.